### PR TITLE
[CI] use obltGitHubComments

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Use https://github.com/elastic/apm-pipeline-library/blob/master/vars/obltGitHubComments.txt

## Why is it important?

Avoid hardcoded strings and easy to normalise the GitHub comments in all the oblt projects

### CODE REVIEW

File: .ci/apm-beats-update.groovy [hunk 1]
```diff
@@ -26,7 +26,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*/run\\s+(?:apm-beats-update\\W+)?.*')
+    issueCommentTrigger('(?i)^/run\\s+(?:apm-beats-update\\W+)?.*')
```
Comment (cachedout) [inline .ci/apm-beats-update.groovy:29]: Is this change related somehow to the migration to `obltGitHubComments` or is it a separate fix? It seems as if it's separate and so perhaps we should update the description of this PR?

**Review by cachedout — APPROVED** [review]

File: .ci/apm-beats-update.groovy (hunk 1 above)
Comment (v1v) [reply .ci/apm-beats-update.groovy:29]: Unrelated but forcing to use `/run` rather than `bla bla bla /run`. So I'll revert it to keep the PR clean

Comment (v1v) [reply .ci/apm-beats-update.groovy:29]:
```suggestion
    issueCommentTrigger('(?i).*/run\\s+(?:apm-beats-update\\W+)?.*')
```

**Review by kuisathaverat — APPROVED** [review]

**Comment by v1v** [comment]
Unrelated failures with this PR, since it only affects the pipeline itself

**Pull request merged**